types/react/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "dependencies": {
-    "csstype": "^3.0.2"
+    "csstype": "^3.0.2",
+    "definitely-typed": "file:../.."
```

Comment:
Result of running `npm install --prefix types/react`
Because of the following error:
```
A module look-up failed, this often occurs when you need to run `npm install` on a dependent module before you can lint.

Before you debug, first try running:
    npm install --prefix C:/Users/Avasam/Documents/Git/DefinitelyTyped/types/react

Then re-run. Full error logs are below.

Errors in typescript@4.6 for external dependencies:

    at testTypesVersion (C:\Users\Avasam\Documents\Git\DefinitelyTyped\node_modules\@definitelytyped\dtslint\bin\src\index.js:189:15)
    at async runTests (C:\Users\Avasam\Documents\Git\DefinitelyTyped\node_modules\@definitelytyped\dtslint\bin\src\index.js:165:13)
    at async main (C:\Users\Avasam\Documents\Git\DefinitelyTyped\node_modules\@definitelytyped\dtslint\bin\src\index.js:77:9)

added 1 package, and audited 3 packages in 688ms
```